app/components/ui/Footer.jsx

```diff
@@ -4,15 +4,44 @@ import { Link } from 'react-router';
 function Footer() {
   return (
     <footer className="site-footer" role="contentinfo">
-      <div className="footer-content">
-        <div className="footer-text">
-            ©{ new Date().getFullYear() } Shelter Tech
+      <div className="site-footer__content">
+        <div className="site-footer__top">
+          <section className="service-guide">
+            <div className="service-guide__icon" />
+            <h1 className="service-guide__text">
+              SF Service Guide
```

Comment:
I think we want this to conditionally say "Ask Darcel" or "SF Service Guide" depending on what domain name the site is being served from. Here's an example of where we're currently doing a check based on the subdomain: https://github.com/ShelterTechSF/askdarcel-web/blob/861898cbc87b5e6459791b0ecbb3b6cf1cbea791/app/pages/HomePage.jsx#L43

Reply:
Will do